ttnn/cpp/ttnn/operations/matmul.hpp

```diff
@@ -78,13 +78,28 @@ inline ttnn::Tensor matmul(
 	user_core_coord = CoreCoord(core_grid->x, core_grid->y);
     }
     auto output_tensor = tt::operations::primary::matmul(
-        input_tensor_a_4d, input_tensor_b_4d, std::nullopt /*bias*/, program_config, memory_config, dtype, compute_kernel_config, false /*untilize_out*/,  user_core_coord, input_b_is_batched);
+        input_tensor_a_4d, input_tensor_b_4d, /*bias=*/std::nullopt, program_config, memory_config, dtype, compute_kernel_config, /*untilize_out=*/false,  user_core_coord, /*user_fused_activation=*/std::nullopt, input_b_is_batched);
     while (output_tensor.get_shape().rank() != input_tensor_a_shape.rank()) {
         output_tensor = ttnn::squeeze_from_4D(output_tensor, input_tensor_a_shape.rank());
     }
     return output_tensor;
 }
 
+std::optional<UnaryWithParam> get_fused_activation(const std::optional<const std::string>& activation) {
```

Comment:
You should use `string_to_unary_with_param` function from `eltwise_unary_op.hpp`

Reply:
I used the start of that as this method. My concern is that there are a lot more ops in string_to_unary_with_param.

Is it okay if e.g. someone accidentally puts in sign instead of silu and strange stuff happens?

Reply:
Talked offline. Approach is ok. Updated to use string_to_unary_with_param